signing_service/signing_service/signing_service.py

```diff
@@ -570,6 +583,15 @@ def _parse_arguments() -> argparse.Namespace:
     )
     crash_logger.handlers[0].client = raven_client  # type: ignore
 
+    if any('smtp-notifier' in args for args in argparse.ArgumentParser().parse_known_args()):  # type: ignore
+        notifier_class = EmailNotifier(
+            args.from_email_address,
+            args.from_email_password,
+            args.to_email_addresses,
+        )
+    else:
+        notifier_class = ConsoleNotifier()  # type: ignore
```

Comment:
```suggestion
        notifier = ConsoleNotifier()  # type: ignore
```
BTW: why ` # type: ignore`?

Reply:
Incompatible types in assignment (expression has type "ConsoleNotifier", variable has type "EmailNotifier")
I don't know how to fix this in `SigningService` I have `notifier: Union[ConsoleNotifier, EmailNotifier]`, but it doesn't seems to make difference.

Reply:
You could make an abstract class `Notifier`, from which `ConsoleNotifier` and `EmailNotifier` would be derived. And then use type `Notifier` in annotations.

Reply:
That didn't change anything and now I've got:
```
signing_service/signing_service/signing_service.py:295: error: "Notifier" has no attribute "send"
signing_service/signing_service/signing_service.py:304: error: "Notifier" has no attribute "send"
signing_service/signing_service/signing_service.py:594: error: Incompatible types in assignment (expression has type "ConsoleNotifier", variable has type "EmailNotifier")
```
I thing I'm gone stick with #ignore